### PR TITLE
Upgrade GHC 8.0.1 -> 8.0.2 and add GHC 8.2.1 in Travis config

### PR DESCRIPTION
**Pull request by mschristiansen**

### CODE REVIEW

**Comment by mschristiansen** [comment]
The build fails for the GHC 8.2.1 contrained build. Seems to be because of servant-0.11 dependency. Not sure if there is anything to do about that currently(?).

File: .travis.yml [hunk 1]
```diff
@@ -5,17 +5,21 @@ sudo: false
...
     - env: CABALVER=1.18 GHCVER=7.8.4
       addons: {apt: {packages: [cabal-install-1.18,ghc-7.8.4], sources: [hvr-ghc]}}
     # Constrained
-    - env: CABALVER=1.24 GHCVER=8.0.1 CABALCONFIG=servant-0.9
-      addons: {apt: {packages: [cabal-install-1.24,ghc-8.0.1], sources: [hvr-ghc]}}
-    - env: CABALVER=1.24 GHCVER=8.0.1 CABALCONFIG=servant-0.8
-      addons: {apt: {packages: [cabal-install-1.24,ghc-8.0.1], sources: [hvr-ghc]}}
+    - env: CABALVER=1.24 GHCVER=8.2.1 CABALCONFIG=servant-0.9
+      addons: {apt: {packages: [cabal-install-1.24,ghc-8.2.1], sources: [hvr-ghc]}}
+    - env: CABALVER=1.24 GHCVER=8.0.2 CABALCONFIG=servant-0.9
```
Comment (zohl) [inline .travis.yml]: Here should be `CABALCONFIG=servant-0.11`

**Comment by zohl** [comment]
GHC 8.2.1 comes with `base-4.10` and `servant-0.9.*` has constraint `base < 4.10`, so cabal is unable to resolve dependencies.
You should just specify, that in case of this version GHC you want cabal to install `servant-0.11` (`servant-0.10` has the same problem).

I commented where to update the constraint.
Additionally you should create file `.travis/servant-0.11-cabal.config` with corresponding contents to make it work.

**Comment by zohl** [comment]
Thank you! Merged.

**Pull request merged**